content/zh-cn/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -18,120 +18,120 @@ This page explains how to configure your DNS
-[Migrating to CoreDNS](/docs/tasks/administer-cluster/coredns/#migrating-to-coredns)
-explains how to use `kubeadm` to migrate from `kube-dns`.
 -->
 你的集群必须运行 CoreDNS 插件。
-文档[迁移到 CoreDNS](/zh-cn/docs/tasks/administer-cluster/coredns/#migrating-to-coredns)
-解释了如何使用 `kubeadm` 从 `kube-dns` 迁移到 CoreDNS。
+
+{{% version-check %}}
 
 <!-- steps -->
 
 <!-- 
-## Introduction 
+## Introduction
 
 DNS is a built-in Kubernetes service launched automatically
-using the addon manager
-[cluster add-on](http://releases.k8s.io/master/cluster/addons/README.md). 
+using the _addon manager_ [cluster add-on](http://releases.k8s.io/master/cluster/addons/README.md).
 -->
-## 介绍
+## 介绍   {#introduction}
 
-DNS 是使用[集群插件](http://releases.k8s.io/master/cluster/addons/README.md)
-管理器自动启动的内置的 Kubernetes 服务。
-
-<!-- 
-As of Kubernetes v1.12, CoreDNS is the recommended DNS Server, replacing kube-dns.  If your cluster
-originally used kube-dns, you may still have `kube-dns` deployed rather than CoreDNS.
--->
-从 Kubernetes v1.12 开始，CoreDNS 是推荐的 DNS 服务器，取代了 kube-dns。 如果
-你的集群原来使用 kube-dns，你可能部署的仍然是 `kube-dns` 而不是 CoreDNS。
+DNS 是使用 **插件管理器**
+[集群插件](http://releases.k8s.io/master/cluster/addons/README.md)自动启动的 Kubernetes 内置服务。
 
+{{< note >}}
 <!--
 The CoreDNS Service is named `kube-dns` in the `metadata.name` field.  
-This is so that there is greater interoperability with workloads that relied on the legacy `kube-dns` Service name to resolve addresses internal to the cluster. Using a Service named `kube-dns` abstracts away the implementation detail of which DNS provider is running behind that common name.
+The intent is to ensure greater interoperability with workloads that relied on
+the legacy `kube-dns` Service name to resolve addresses internal to the cluster.
+Using a Service named `kube-dns` abstracts away the implementation detail of
+which DNS provider is running behind that common name.
 -->
-{{< note >}}
 CoreDNS 服务在其 `metadata.name` 字段被命名为 `kube-dns`。
 这是为了能够与依靠传统 `kube-dns` 服务名称来解析集群内部地址的工作负载具有更好的互操作性。
 使用 `kube-dns` 作为服务名称可以抽离共有名称之后运行的是哪个 DNS 提供程序这一实现细节。
 {{< /note >}}
 
 <!--
-If you are running CoreDNS as a Deployment, it will typically be exposed as a Kubernetes Service with a static IP address.
-The kubelet passes DNS resolver information to each container with the `-cluster-dns=<dns-service-ip>` flag.
+If you are running CoreDNS as a Deployment, it will typically be exposed as
+a Kubernetes Service with a static IP address.
+The kubelet passes DNS resolver information to each container with the
+`--cluster-dns=<dns-service-ip>` flag.
 -->
 如果你在使用 Deployment 运行 CoreDNS，则该 Deployment 通常会向外暴露为一个具有
 静态 IP 地址 Kubernetes 服务。
 kubelet 使用 `--cluster-dns=<DNS 服务 IP>` 标志将 DNS 解析器的信息传递给每个容器。
 
 <!-- 
 DNS names also need domains. You configure the local domain in the kubelet
-with the flag `-cluster-domain=<default-local-domain>`. 
+with the flag `--cluster-domain=<default-local-domain>`.
 -->
 DNS 名称也需要域名。 你可在 kubelet 中使用 `--cluster-domain=<默认本地域名>`
```

Comment:
Remove redundant space.
```suggestion
DNS 名称也需要域名。你可在 kubelet 中使用 `--cluster-domain=<默认本地域名>`
```

Maybe we can change `。` to `，` in sentences like this.